examples/monolithic_build/Makefile

```diff
@@ -104,17 +104,17 @@ $(LIB1024_FULL): $(MLK_SOURCE)
 $(BIN512_FULL): $(APP_SOURCE) $(LIB512_FULL)
 	$(Q)echo "$@"
 	$(Q)[ -d $(@) ] || mkdir -p $(@D)
-	$(Q)$(CC) $(CFLAGS) -DMLK_CONFIG_FILE="\"config_512.h\"" $(INC) $^ -o $@
+	$(Q)$(CC) $(CFLAGS) -DMLK_CONFIG_API_PARAMETER_SET=512 $(INC) $^ -o $@
```

Comment:
Does this mean the `config_{512,768,1024}.h` headers can be deleted?

Reply:
No, they are needed for the library build.

Reply:
ahh right - makes sense.